tests/Microsoft.Identity.Test.Unit/CryptographyTests.cs

```diff
@@ -34,6 +38,42 @@ public void SignWithCertificate()
             Assert.IsNotNull(value);
             Assert.AreEqual("MrknKHbOAVu2iuLHMFSk2SK773H1ysxaAjAPcTXYSfH4P2fUfvzP6aIb9MkBknjoE_aBYtTnQ7jOAvyQETvogdeSH7pRDPhCk2aX_8VIQw0bjo_zBZj5yJYVWQDLIu8XvbuzIGEvVaXKz4jJ1nYM6toun4tM74rEHvwa0ferafmqHWOd5puPhlKH1VVK2RPuNOoKNLWBprVBaAQVJVFOdRcd3iR0INBHykxtOsG0pgo0Q2uQBlKP7KQb7Ox8i_sw-M21BuUzdIdGs_oeUYh0B8s-eIGf34JmHRWMwWCnRWzZgY9YuIjRoaWNqlWYb8ASjKOxzvk99x8eFEYKOjgAcA", value);
         }
+
+        [TestMethod]
+        [TestCategory("CryptographyTests")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Internal.Analyzers", "IA5352:DoNotMisuseCryptographicApi", Justification = "Suppressing RoslynAnalyzers: Rule: IA5352 - Do Not Misuse Cryptographic APIs in test only code")]
+        public void SignWithNonRsaCertificate()
```

Comment:
For test naming, good practice is to add what the expected result is, so it's easier to see at a glance.
```suggestion
        public void SignWithNonRsaCertificate_ThrowsException()
```